### ISSUE TITLE
Include definition for AboutLibraries

### ISSUE THREAD

**Issue by mueller-ma** [GrenderG/Toasty#137]
AboutLibraries is a library to offer some information of libraries: http://mikepenz.github.io/AboutLibraries
I use this lib to shown information about all used libs in my app.
Can you add a definition file to this repo, so AboutLibraries knows Toasty?
https://github.com/mikepenz/AboutLibraries/wiki/HOWTO:-Contribute
https://github.com/mikepenz/AboutLibraries/wiki/HOWTODEV:-Include-in-your-library

**Comment by GrenderG** [comment]
Will definitely do once I have some time! I'm open to PRs too.

**Comment by nikothegreek** [comment]
Ok, I think I know why AboutLibraries can't find any information:

If you search the databases ([direct search example 1](https://ossindex.sonatype.org/search?q=Toasty), [direct search example 2](https://search.maven.org/search?q=Toasty)), where normally every library should be found, for "Toasty", they don't find anything.

If you e.g. search for "AboutLibraries" ([direct search link](https://ossindex.sonatype.org/search?q=AboutLibraries)) or also for libraries with "com.github" in the beginning like "AppIntro" ([direct search link](https://ossindex.sonatype.org/search?q=AppIntro)), you find their entries.

But I don't know why it can't be found...

**Comment by mueller-ma** [comment]
I don't think that AboutLibraries is searching for dependencies online. You have to write a definition file and include it in the library or in AboutLibraries: https://github.com/mikepenz/AboutLibraries/wiki/HOWTODEV:-Include-in-your-library

**Comment by nikothegreek** [comment]
Oh, you're right. Otherwise it wouldn't work offline....

I [found a PR](https://github.com/mikepenz/AboutLibraries/pull/507) on AboutLibraries repo where somebody wanted to add a new library to AboutLibraries ([you can add a library directly in AboutLibraries or in your own app](https://github.com/mikepenz/AboutLibraries/wiki/HOWTODEV:-Include-in-your-library)).
But Mike said with the new update it is no longer needed because AboutLibraries searches for all the needed information in a POM file.

If you e.g. search for POM in another library ([Example](https://github.com/AppIntro/AppIntro/search?q=pom&unscoped_q=pom)), you will find the information we need.
If you search for it in Toasty repo, nothing appears to have "POM" included.

I would say we don't include it like the wiki entry you related to in the last comment but do it with a POM file.

**Comment by mueller-ma** [comment]
Good to know. I opened #512, so that the docs get updated.

**Comment by nikothegreek** [comment]
Perfect. Otherwise you can manually add the Toasty information in the openhab app.
